dashboard/test/models/census/census_summary_test.rb

```diff
@@ -389,7 +417,7 @@ def compute_teaches_cs(args)
 
   def validate_explanation(explanation)
     assert_not_nil explanation
-    assert_equal 8, explanation.length, explanation
+    assert_equal 9, explanation.length, explanation
```

Comment:
Is there any way to deduce this number rather than having a constant?

Reply:
I can't think of a way. It is the expected number of fields in the explanation. Since that is something we build up one piece at a time, I don't see a way to deduce it.